README.md

```diff
@@ -76,8 +76,9 @@ render(<Picker />, mountNode);
 | getPopupContainer | function(trigger) |  | to set the container of the floating layer, while the default is to create a div element in body |
 | onChange | Function(date: moment, dateString: string) |  | a callback function, can be executed when the selected time is changing |
 | onOpenChange | Function(open:boolean) |  | called when open/close picker |
-| onFocus | (evnet:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on focus |
-| onBlur | (evnet:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on blur |
+| onFocus | (event:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on focus |
+| onBlur | (event:React.FocusEventHandler<HTMLInputElement>) => void |  | called like input's on blur |
+| onKeyDown | (event:React.KeyboardEventHandler<HTMLInputElemet>, preventDefault:void) => void |  | input on keydown event |
```

Comment:
```suggestion
| onKeyDown | (event:React.KeyboardEventHandler<HTMLInputElemet>, preventDefault: () => void) => void |  | input on keydown event |
```

the type of `preventDefault` should be function rather than `void`

Reply:
i think, 
i wrote wrong event there, it should be ```React.KeyboardEvent<HTMLInputElement>```